marathon_envoy_poc/app.py

```diff
@@ -181,12 +186,78 @@ def endpoints():
 @flask_app.route("/v2/discovery:listeners", methods=["POST"])
 def listeners():
     # TODO: Without TLS/SNI stuff, this is largely static
-    pass
+    filter_chains = [
+        FilterChain([
+            Filter("envoy.http_connection_manager",
+                   HttpConnectionManager("http", "http", own_config_source()))
+        ])
+    ]
+    listeners = [
+        Listener(
+            "http",
+            flask_app.config["HTTP_LISTEN_ADDR"],
+            flask_app.config["HTTP_LISTEN_PORT"],
+            filter_chains
+        )
+    ]
+
+    return jsonify(DiscoveryResponse("0", listeners, TYPE_LDS))
```

Comment:
I assume we'll want to use an incrementing version here once we have something less static.